### ISSUE TITLE
Unit tests printing debugging information

### ISSUE THREAD

**Issue by mattngc** [cherab/core#94]
Someone has left some debugging output in their unit tests. Running unit tests for CHERAB results in a lot of numbers being printed to the terminal. These should be removed, detailed testing output should only be printed if requested through a flag of some kind.

**Comment by jacklovell** [comment]
It's in the 2D interpolators tests:
```
jlovell@freia010 cherab_core> grep -r print --include="*test*"
Binary file cherab/core/math/interpolators/tests/__pycache__/test_interpolators2d.cpython-37.pyc matches
cherab/core/math/interpolators/tests/test_interpolators2d.py:                        print(x, y, x_order, y_order, r, v, abs(r - v))
```

**Comment by mattngc** [comment]
Fixed by @jacklovell in PR #99.
